lib/Sema/CSSimplify.cpp

```diff
@@ -9746,6 +9747,16 @@ performMemberLookup(ConstraintKind constraintKind, DeclNameRef memberName,
   if (auto *selfTy = instanceTy->getAs<DynamicSelfType>())
     instanceTy = selfTy->getSelfType();
 
+  // Dynamically isolated function types have a magic '.isolation'
+  // member that extracts the isolation value.
+  if (auto *fn = dyn_cast<FunctionType>(instanceTy)) {
```

Comment:
Maybe better use `instanceTy->getAs<FunctionType>()` because member could be wrapped in parens